Package.swift

```diff
@@ -86,7 +86,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/objecthub/swift-numberkit.git", from: "2.4.1"),
         .package(url: "https://github.com/thebarndog/swift-dotenv.git", from: "1.0.0"),
-        .package(url: "https://github.com/grpc/grpc-swift.git", from: "1.16.0"),
+        .package(url: "https://github.com/grpc/grpc-swift.git", from: "1.23.0"),
```

Comment:
Not sure why this was needed to build but old version causes build failures.. everything is still passing with this upgrade though so it should be fine